### PR TITLE
updated autoinstrumentation description to match other languages

### PR DESCRIPTION
**Pull request by sy-bee**
And be more precise in wording to indicate that instrumentation libraries are shipped rather than detected.

This addresses an issue described here: https://github.com/open-telemetry/opentelemetry-operator/issues/2559

### CODE REVIEW

File: content/en/docs/kubernetes/operator/automatic.md [hunk 1]
```diff
@@ -361,11 +361,11 @@ spec:
 
 #### Excluding auto-instrumentation {#python-excluding-auto-instrumentation}
 
-By default the Python auto-instrumentation will detect the packages in your
-Python service and instrument anything it can. This makes instrumentation easy,
-but can result in too much or unwanted data. If there are any packages you do
-not want to instrument, you can set the `OTEL_PYTHON_DISABLED_INSTRUMENTATIONS`
-environment variable
+By default, the Python auto-instrumentation ships with
+[many instrumentation libraries]([https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/config.md#instrumentations](https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/python/requirements.txt)).
```
Comment (svrnm) [inline content/en/docs/kubernetes/operator/automatic.md:365]: here something went wrong, probably you only wanted to have the last URL?

```suggestion
[many instrumentation libraries](https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/python/requirements.txt).
```

**Comment by svrnm** [comment]
thank you @sy-bee

File: content/en/docs/kubernetes/operator/automatic.md (hunk 1 above)
Comment (sy-bee) [reply content/en/docs/kubernetes/operator/automatic.md]: Thank you! Need to work on my copy/paste skills! 😁

Comment (svrnm) [reply content/en/docs/kubernetes/operator/automatic.md]: same, happens to me sooo often ;-)

**Review by swiatekm-sumo — APPROVED** [review]

File: content/en/docs/kubernetes/operator/automatic.md [hunk 2]
```diff
@@ -361,11 +361,11 @@ spec:
...
-By default the Python auto-instrumentation will detect the packages in your
-Python service and instrument anything it can. This makes instrumentation easy,
-but can result in too much or unwanted data. If there are any packages you do
-not want to instrument, you can set the `OTEL_PYTHON_DISABLED_INSTRUMENTATIONS`
-environment variable
+By default, the Python auto-instrumentation ships with
+[many instrumentation libraries](https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/python/requirements.txt).
+This makes instrumentation easy, but can result in too much or unwanted data. If
+there are any packages you do not want to instrument, you can set the
+`OTEL_PYTHON_DISABLED_INSTRUMENTATIONS` environment variable
```
Comment (svrnm) [inline content/en/docs/kubernetes/operator/automatic.md:368]:
```suggestion
`OTEL_PYTHON_DISABLED_INSTRUMENTATIONS` environment variable.
```

**Review by TylerHelmuth — APPROVED** [review]

**Review by jaronoff97 — APPROVED** [review]

**Review by cartermp — APPROVED** [review]

**Pull request merged**